Yoast/wordpress-seo
# Fix jQuery not a function

## Summary
This PR can be summarized in the following changelog entry:

* Fixes jQuery assignment after https://github.com/Yoast/wordpress-seo/pull/13692

## Relevant technical choices:

* There was a missing assignment for the `$` alias, added `const $ = jQuery;`

## Test instructions
- to reproduce the error on trunk: activate the Classic Editor plugin
- edit a post
- see the meta box doesn't render
- console error in Chrome: `classicEditorData.js?7ad1:101 Uncaught TypeError: $ is not a function`
- switch to this branch and build
- edit a post (maybe refresh)
- see the meta box renders correctly
- test setting / removing a featured image and check everything works as expected

## UI changes

## Documentation

## Quality assurance
